scripts/ci/docker_upgrade.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+set -e
+
+check_parameters() {
+  echo "-- check_parameters"
+
+  if [ -z "${IMAGE_NAME}" ] || [ -z "${IMAGE_VARIANT}" ] || [ -z "${DOCKER_REPOSITORY}" ]; then
+    echo "\${IMAGE_NAME}, \${IMAGE_VARIANT} and \${DOCKER_REPOSITORY} need to be defined."
+
+    false
+  fi
+}
+
+check_version() {
+  echo "-- check_version"
+
+  git fetch
+  git branch
+  DOCKERFILE="scripts/docker/${IMAGE_NAME}/Dockerfile.${IMAGE_VARIANT}"
+  PREV_VERSION=$(git diff origin/develop HEAD -- "${DOCKERFILE}" | grep -e "-LABEL version=\".*\"" | awk -F"\"" '{ print $2 }')
+  NEXT_VERSION=$(git diff origin/develop HEAD -- "${DOCKERFILE}" | grep -e "+LABEL version=\".*\"" | awk -F"\"" '{ print $2 }')
+
+  if [ -z "${NEXT_VERSION}" ]; then
+    echo ""
+    echo "ERROR: No version label defined in Dockerfile. You may need to add 'LABEL version' in '${DOCKERFILE}'. Aborting."
```

Comment:
I'd print error messages to stderr:
```suggestion
    >&2 echo "ERROR: No version label defined in Dockerfile. You may need to add 'LABEL version' in '${DOCKERFILE}'. Aborting."
```